### PR TITLE
feat: vipps for parceferme

### PR DESCRIPTION
**Pull request by danielgent**
The tests pass at least!

In order to try this out we need to add the `vipps`  config to the default tenant and parcferme.

Then it should _just work_ right? As we're already logging into `vipps` through Auth2 :thinking:

### CODE REVIEW

File: test/fixtures/vendorSettings.ts [hunk 1]
```diff
@@ -73,3 +73,21 @@ export const SESAMY_VENDOR_SETTINGS: VendorSettings = {
...
+    primaryHoverColor: "#002E62",
+  },
+  loginBackgroundImage:
+    "https://assets.sesamy.com/vendors/parcferme/SNH_5894.jpeg",
+  checkoutHideSocial: false,
+  supportEmail: null,
+  supportUrl: "https://www.parcferme.no",
+  siteUrl: "https://www.parcferme.no",
+  termsAndConditionsUrl: null,
+};
```
Comment (danielgent) [inline test/fixtures/vendorSettings.ts:93]: copy-pasted from prod

File: src/utils/components/VippsLogo.tsx
Comment (danielgent) [inline src/utils/components/VippsLogo.tsx:1]: copy-pasted from login2

File: test/integration/__image_snapshots__/social-sign-on-buttons-spec-ts-test-integration-social-sign-on-buttons-spec-ts-should-show-vipps-for-parcferme-as-entered-as-connection-1-snap.png
Comment (danielgent) [inline test/integration/__image_snapshots__/social-sign-on-buttons-spec-ts-test-integration-social-sign-on-buttons-spec-ts-should-show-vipps-for-parcferme-as-entered-as-connection-1-snap.png:1]: voila.


I wouldn't mind snapshot at more resolutions. If we go taller then the buttons stack. If we go narrower then we get the mobile layout.

Comment (danielgent) [reply test/integration/__image_snapshots__/social-sign-on-buttons-spec-ts-test-integration-social-sign-on-buttons-spec-ts-should-show-vipps-for-parcferme-as-entered-as-connection-1-snap.png:1]: Not just for this test but throughout the whole repo!

**Review by markusahlstrand — APPROVED** [review]
Looks good. Should be fine to just add vipps to the publishers tenant and not to have in the default tenant

**Pull request merged**